docs/docs/examples/workflow/multi_step_query_engine.ipynb

```diff
@@ -0,0 +1,493 @@
+{
```

Comment:
<p>"The answer from sub-query-1 is passed into sub-query-2" for the given examples above, that doesn't really make sense. How does it get passed in? Or what does that mean?</p><p> </p><p>Reading further down and reading the code, it seems like steps are actually generated one at a time, and so the answer to the previous question is used to write the next question (if any)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/15438/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Yeah. I think I communicated it wrongly. I will re-write it.